src/main/java/VASSAL/build/module/properties/NumericPropertyPrompt.java

```diff
@@ -43,7 +43,8 @@ public String getNewValue(String oldValue) {
     String s = null;
     do {
       s = (String) JOptionPane.showInputDialog(dialogParent, promptText, null, JOptionPane.QUESTION_MESSAGE, null, null, oldValue);
-    } while (s != null && !isValidValue(s));
+    }
+    while (s != null && !isValidValue(s));
```

Comment:
Let's keep the `while` on the same line as the closing brace.

Reply:
I reverted this, we will also activate the rule "RightCurly on same line" for "while" later on and catch the other occurrences if there are any.